fairing/config.py

```diff
@@ -62,39 +64,65 @@ def reset(self):
         self._deployer_kwargs = {}
 
     def set_preprocessor(self, name=None, **kwargs):
+        """
+
+        :param name: preprocessor name(Default value = None)
+        :param **kwargs: 
```

Comment:
I hesitated if need to remove this, not sure what best practice for kwargs.

Reply:
lets remove it